### ISSUE TITLE
Weighted chances upon encounter

### ISSUE THREAD

**Issue by TurnipGuy30** [Pokemon-PythonRed/Pokemon-PythonRed#100]
```
Current Location: Route 1 (North)

[w] - Go to Viridian City (South)
[s] - Go to Route 1 (South)

> s
Traceback (most recent call last):
  File "c:\[redacted]\Pokemon-PythonRed\app\main.py", line 1176, in <module>
    with get_encounter('route1-s', 'tall-grass')[0] as encounter:
  File "c:\[redacted]\Pokemon-PythonRed\app\main.py", line 643, in get_encounter
    return choices(pokemon, weights)[0]
  File "C:\Python310\lib\random.py", line 521, in choices
    cum_weights = list(_accumulate(weights))
TypeError: unsupported operand type(s) for +: 'dict' and 'float'

C:\[redacted]>
```

_Originally posted by @TurnipGuy30 in https://github.com/Pokemon-PythonRed/Pokemon-PythonRed/issues/98#issuecomment-1345776756_

**Comment by nasserkessas** [comment]
I pulled to the latest commit and I am now getting this error as well. This means that one of the more recent commits caused the problem.

**Comment by nasserkessas** [comment]
All fixed now as of commit [ee7ab81](https://github.com/Pokemon-PythonRed/Pokemon-PythonRed/pull/98/commits/ee7ab819b57de5dce0d5c66881ac3c56a530f472) on branch parcel-delivery

**Comment by TurnipGuy30** [comment]
This was my own fault, sorry.
